tools/runfiles/runfiles.rs

```diff
@@ -19,13 +19,13 @@
 //!     use runfiles::Runfiles;
 //!     ```
 //!
-//! 3.  Create a Runfiles object and use rlocation to look up runfile paths:
+//! 3.  Create a Runfiles object and use Rlocation! to look up runfile paths:
```

Comment:
Should it be title-case? I was under the impression that naming conventions dictated that since it's a function, it should be `rlocation`

Reply:
sure, I wasn't sure what the convention was for macros but I think you're right